Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use alpine for a smaller image size and install only the required packages
```

Comment:
This comment is out of date.  We've been using buster, not alpine, and you've switching it to bullseye now.

Reply:
This was part of the initial template. I'll remove it as its a bit confusing.